File: Practica-6/ejercicio8.c
#include <stdio.h>
int main()
{
    float m1, m2, d, G = 9.81, F;
    printf("Ingrese la masa 1 (gramos): ");
    scanf("%f", &m1);
    printf("Ingrese la masa 2 (gramos): ");
    scanf("%f", &m2);
    printf("Ingrese la distancia (centimetros): ");
    scanf("%f", &d);
    F = G * m1 * m2 / (d*d);
    printf("La fuerza de atraccion expresada en dinas es igual a: %f", F);
}

File: Practica-6-py/ejercicio8.py
m1 = float(input("Ingrese el valor de masa 1 (gramos) -> "))
m2 = float(input("Ingrese el valor de masa 2 (gramos) -> "))
d = float(input("Ingrese el valor de distancia (centímetros) -> "))
g = 9.8
F = (g * m1 * m2) / pow(d, 2)
print("La fuerza de atracción expresada en dinas es igual a: ", F)
